packages/compiler/src/__tests__/__snapshots__/typescript.test.ts.snap

```diff
@@ -747,34 +551,34 @@ Object {
     },
     "inheritedFrom": undefined,
     "kind": "method",
-    "name": "numberAndString",
-    "signatures": Array [
-      Object {
-        "documentation": Object {
-          "contents": Array [
-            "<p>Exported function.</p>
+    "name": "MyFunc",
```

Comment:
these functions are still documented correctly, but their order got moved around in the snapshot